sigmastate/src/main/scala/org/ergoplatform/ErgoLikeContext.scala

```diff
@@ -111,6 +111,18 @@ class ErgoLikeContext(val lastBlockUtxoRoot: AvlTreeData,
   */
   val self: ErgoBox = boxesToSpend(selfIndex)
 
+  /** Current version of the ErgoTree executed by the interpreter.
+    * This property is used to implement version dependent operations and passed to
+    * interpreter via [[specia.sigma.Context]].
+    * The value cannot be assigned on [[ErgoLikeContext]] construction and must be
+    * attached using [[withErgoTreeVersion()]] method.
+    * When the value is None, the [[InterpreterException]] is thrown by the interpreter.
+    */
+  val currentErgoTreeVersion: Option[Byte] = None
```

Comment:
Naming not ideal, as "current" does not say much, should it be something like "executedErgoTreeVersion" ?

Reply:
"executed" is only creates an illusion of better description. Which execution? What are the assumptions? 
In such cases is better to read the docs and "current", by being general enough, encourage the reader to consult the docs.